hack/lib/mesh_resources/peerauthentication.yaml

```diff
@@ -6,93 +6,3 @@ metadata:
 spec:
   mtls:
     mode: STRICT
----
-apiVersion: security.istio.io/v1beta1
-kind: PeerAuthentication
-metadata:
-  name: default
-  namespace: knative-serving
-spec:
-  mtls:
-    mode: STRICT
----
-apiVersion: security.istio.io/v1beta1
-kind: PeerAuthentication
-metadata:
-  name: default
-  namespace: knative-eventing
-spec:
-  mtls:
-    mode: STRICT
----
-apiVersion: security.istio.io/v1beta1
-kind: PeerAuthentication
-metadata:
-  name: default
-  namespace: eventing-e2e0
```

Comment:
Just a note. `eventing-e2e0` - `eventing-e2e4` are namespaces used by upgrade tests. These tests do not run by default in CI with Service Mesh. We might need to define some rules for these namespaces as well.

Reply:
The working theory is, that adding it to `istio-system` will apply it to the whole mesh automatically. At least upstream istio does that and documents it. I'll ask the SM team if we can also use that.